Sprint-1/destructuring/exercise-1/exercise.js

```diff
@@ -6,7 +6,7 @@ const personOne = {
 
 // Update the parameter to this function to make it work.
 // Don't change anything else.
-function introduceYourself(___________________________) {
+function introduceYourself( { name, age, favouriteFood }=personOne) {
```

Comment:
Nit: Try to keep spacing consistent throughout your code. It makes it easier to read and maintain. 🙂
`function introduceYourself( { name, age, favouriteFood } = personOne )`